File: app/state/state.py
from dotenv import load_dotenv
import os
from ..utils.logger import create_logger
load_dotenv()
from asyncio import Queue


bot_number = os.getenv("BOT_NUMBER") + "@s.whatsapp.net"
port = os.getenv("PORT")
key = os.getenv("KEY")
clients = {}
logger = create_logger("test", 10)
pending_feedback: dict[str, Queue] = {}
node_path = os.getenv("NODE_PATH")

File: src/state/state.js
import dotenv from "dotenv";
import { resolve } from "path";
dotenv.config({
  path: resolve(process.cwd(), ".env"),
});
import { CacheMessage } from "../whatsapp_bot/utils/caching.js";


export const cache = new CacheMessage(process.env.MESSAGE_CACHE_LIMIT || 500);
export const botNumber = (process.env.BOT_NUMBER || "") + "@s.whatsapp.net";
export const port = process.env.PORT || "8765";
export const key = process.env.KEY;
export const authStateFile = process.env.AUTH_STATE_FILE;
